src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshotManager.cs

```diff
@@ -146,9 +144,11 @@ public override void DocumentAdded(HostProject hostProject!!, HostDocument docum
                 }
                 else
                 {
+#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
```

Comment:
As above, I don't like these suppressions, particularly knowing its calling into roslyn. If something is non-nullable in the roslyn API you can assume that it will throw a NRE if you pass a null in.